src/tests/unit/tests/common/components/visualization-toggle.test.tsx

```diff
@@ -1,36 +1,47 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import { shallow } from 'enzyme';
+import { Toggle } from '@fluentui/react';
+import { render } from '@testing-library/react';
+import { userEvent } from '@testing-library/user-event';
 import * as React from 'react';
-import { IMock, Mock, Times } from 'typemoq';
-
+import { IMock, Mock, It } from 'typemoq';
 import {
     VisualizationToggle,
     VisualizationToggleProps,
 } from '../../../../../common/components/visualization-toggle';
+import {
+    expectMockedComponentPropsToMatchSnapshots,
+    mockReactComponents,
+    useOriginalReactElements,
+} from '../../../mock-helpers/mock-module-helpers';
+jest.mock('@fluentui/react');
 
 describe('VisualizationToggleTest', () => {
+    mockReactComponents([Toggle]);
     test('render with no label', () => {
         const generatedProps = generateVisualizationToggleProps();
-        const renderedToggle = shallow(<VisualizationToggle {...generatedProps} />);
-        expect(renderedToggle.getElement()).toMatchSnapshot();
+        const renderResult = render(<VisualizationToggle {...generatedProps} />);
+        expect(renderResult.asFragment()).toMatchSnapshot();
+        expectMockedComponentPropsToMatchSnapshots([Toggle]);
```

Comment:
Since we test that `onClick` works as expected later and the other props are rendered properly in the regular snapshot, we don't need to snapshot the props.
```suggestion
```

Reply:
Made the change